src/main/java/dev/snowdrop/jira/atlassian/model/Release.java

```diff
@@ -1,50 +1,28 @@
 package dev.snowdrop.jira.atlassian.model;
 
+import com.fasterxml.jackson.annotation.JsonIgnore;
+import com.fasterxml.jackson.annotation.JsonProperty;
+
 import java.util.List;
 
 public class Release {
-	private String jiraProject;
```

Comment:
How will you been able to create a JIRA issue within the target JIRA project without this field ?